drltrace/drltrace.c

```diff
@@ -197,7 +197,7 @@ iterate_exports(const module_data_t *info, bool add)
                 IF_DEBUG(bool ok =)
                     drwrap_wrap_ex(func, lib_entry, NULL, (void *) sym->name, 0);
                 ASSERT(ok, "wrap request failed");
-                NOTIFY(2, "wrapping export %s!%s @"PFX""NL,
+                NOTIFY(2, "wrapping export %s!%s @"PFX"NL,
```

Comment:
This looks like a syntax error: " is unmatched.  Both "" needed to be removed.  Here and above.